### PR TITLE
customArray created

### PR DESCRIPTION
**Pull request by romaanankin**
Please, write some comments on my code

### CODE REVIEW

File: src/hometasks/CustomArrayTask/CustomArray.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+    public boolean isEmpty() {
+        return arraySize == 0;
+    }
+
+    public Object get(int index) {
+        return data[index];
+    }
+
```
Comment (YatKat) [inline src/hometasks/CustomArrayTask/CustomArray.java]: check your index to sure that it is not out of bound

File: src/hometasks/CustomArrayTask/Main.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        System.out.println(customArray.get(0));
+        System.out.println(customArray.get(1));
+        System.out.println(customArray.get(2));
+        System.out.println(customArray.get(3));
+        System.out.println(customArray.get(3));
+
+        customArray.clear();
+        System.out.println(customArray.size());
+    }
+}
```
Comment (YatKat) [inline src/hometasks/CustomArrayTask/Main.java]: add space

File: src/hometasks/CustomArrayTask/CustomArray.java [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+        if (data.length < capacity) {
+            Object[] tmp = new Object[data.length * 2];
+            for (int i = 0; i < data.length; i++) {
+                tmp[i] = data[i];
+            }
+            data = tmp;
+        }
+    }
+
+    public void checkIndex(int index) {
```
Comment (boroda4436) [inline src/hometasks/CustomArrayTask/CustomArray.java]: Make this method private

File: src/hometasks/CustomArrayTask/CustomArray.java [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+    public boolean removeAll(Collection c) {
+        return false;
+    }
+
+    @Override
+    public boolean containsAll(Collection c) {
+        return false;
+    }
+
+}
```
Comment (boroda4436) [inline src/hometasks/CustomArrayTask/CustomArray.java]: Empty line at the EOF is required

**Pull request merged**